Silicon/CommonSocPkg/Include/MeChipset.h

```diff
@@ -0,0 +1,33 @@
+/** @file
+  Chipset definition for ME Devices.
+
+ Copyright (c) 2020, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+#ifndef _ME_CHIPSET_H_
+#define _ME_CHIPSET_H_
+
+#define ME_BUS                            0
+#define ME_DEVICE_NUMBER                  22
+#define HECI_FUNCTION_NUMBER              0

```

Comment:
ME device number is same for client platforms but its different for SPS5.

Reply:
Same as MeBiosPayloadData.h, a silicon specific MeChipset.h will be used from the silicon specific directory with proper ME device number.

Reply:
Can we use PCD for the device number, so platforms does not have to change this.

Reply:
HECI_DEVICE mapping is also different from SPS5. So, eventually MeChipset.h needs to be overridden for PCI mapping and HECI device mapping. 
```
- APL/CFL/CML/etc.
Silicon/CommonSocPkg/Include/MeChipset.h
- SPS5
Silicon/<Sps5 Specific SocPkg>/Include/MeChipset.h
```